azext_iot/common/fileops.py

```diff
@@ -0,0 +1,47 @@
+from os import makedirs, remove, listdir
```

Comment:
module is missing headers (coding utf8 (less important for py3 but lets be consistent & explicit) and license.